packages/ui-components/src/components/layout/sidebar/menu/group/Group.vue

```diff
@@ -59,17 +66,20 @@
 </template>
 
 <script setup lang="ts">
-import ArrowFilled from '~~/src/components/layout/sidebar/menu/group/ArrowFilled.vue'
-import Plus from '~~/src/components/layout/sidebar/menu/group/Plus.vue'
+import Plus from '~~/src/components/global/icon/Plus.vue'
+import Edit from '~~/src/components/global/icon/Edit.vue'
+import ArrowFilled from '~~/src/components/global/icon/ArrowFilled.vue'
 import CommonBadge from '~~/src/components/common/Badge.vue'
 
 defineProps<{
   tag?: string
   title?: string
   collapsible?: boolean
   collapsed?: boolean
-  plusText?: string
-  plusClick?: () => void
+  icon?: 'add' | 'edit'
```

Comment:
Also not a blocker; but we can probably just make it accept any icon, like how buttons for example do

Reply:
Yeah, this can definitely be improved, I'll do it in another branch though